extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java

```diff
@@ -27,6 +27,35 @@ public static String removeNonDigitCharacters(String toRemove) {
         return toRemove.replaceAll("\\D+", "");
     }
 
+    /**
+     * <p>Convert a mixed number word to a long.</p>
+     * <p>Examples:</p>
+     * <ul>
+     *     <li>123 -&gt; 123</li>
+     *     <li>1.23K -&gt; 1230</li>
+     *     <li>1.23M -&gt; 1230000</li>
+     * </ul>
+     * @param numberWord string to be converted to a long
+     * @return a long
+     * @throws NumberFormatException
+     * @throws ParsingException
+     */
+    public static long mixedNumberWordToLong(String numberWord) throws NumberFormatException, ParsingException {
```

Comment:
Since this is a public method you could add a test only for this one

Reply:
Done. just noticed that there are no tests for the util methods.